python_ta/transforms/ExprWrapper.py

```diff
@@ -1,4 +1,4 @@
-from typing import Dict, List, Optional, Union
+from typing import Optional, Union, dict, list
```

Comment:
`dict` and `list` do not need to be imported; but, a `__future__` import does need to be added to the top of this file